### ISSUE TITLE
operator invitation

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#829]
Add a link to the how-to-issue wiki page in the operator invitation emails

**Comment by dustymc** [comment]
and put the instructions for filing issues on the authenticate page

**Comment by mkoo** [comment]
+1

**Comment by dustymc** [comment]
dropping back to needs discussion - I need a stable target

**Comment by dustymc** [comment]
I still need this.

https://github.com/ArctosDB/documentation-wiki/wiki/How-to-Use-Github-for-Arctos exists but is a mess - there are several dead links, and I can't find a link to https://github.com/ArctosDB/documentation-wiki/wiki/How-to--Use-Issues-in-Arctos. I think we need to somehow figure out stable URLs - switch software, establish foolproof procedures, SOMETHING - before linking this from Arctos. I'm elevating priority - a linkable participation guide for new users is critical.

**Comment by mkoo** [comment]
I hear ya-- update: I'm reviewing a lot of documentation frameworks that we
can migrate these wiki pages to. I've narrowed it down to 1 or 2 using
jekyll (most likely). Now I need to carve out time to do this move with
Philip. We will have time to tackle/ focus on in June once a few deadlines
are met and he's back from vacation. Keep bugging me but on my priority
radar!

**Comment by campmlc** [comment]
I just went in and fixed the links to the Arctos Documentation and How To
Guides, which were referencing the old wiki site, not the
documentation-wiki. I didn't find any other broken links? Everything I
clicked on seemed functional. Could most of these issues be related to the
incomplete transfer from arctos-wiki?
 Happy to have this moved to a new portal as long as we can still have
editing access.
Do we have a timeline for when this can be released to members at large?
Sooner the better.

**Comment by campmlc** [comment]
Just added the link for How to Use Issues in Arctos to
https://github.com/ArctosDB/documentation-wiki/wiki/How-to-Use-Github-for-Arctos
.

Dusty, let me know what other problems you see.

**Comment by dustymc** [comment]
That fixes my immediate needs - thanks!

I also grabbed a DOI (http://dx.doi.org/10.7299/X75B02M5) for the intro page, so if it does move we (the ezid/ucb-mvz account) can just edit the DOI record instead of changing anything in Arctos.

I added

>See <a href="http://dx.doi.org/10.7299/X75B02M5">http://dx.doi.org/10.7299/X75B02M5</a> for Arctos resources.

to the operator invitation email and authentication form - suggestions for better verbiage appreciated.

Closing this, the general conversation should continue in #837.
